kuma/static/styles/components/wiki/promos.scss

```diff
@@ -150,8 +160,10 @@
 // max 768px
 @media #{$mq-mobile-and-down} {
     #wiki-promo-sauce-labs-a {
-        @include bidi-style(background-position, bottom 20px center, background-position, bottom 20px center);
-        @include bidi-style(padding, 20px 10px 140px, padding, 20px 10px 140px);
+        @include bidi((
+            (background-position, bottom 20px center, background-position, bottom 20px center),
+            (padding, 20px 10px 140px, padding, 20px 10px 140px)
```

Comment:
Nit: If both the LTR and RTL style rules are the same, is there a specific reason we should specify it inside `bidi`?

Reply:
Yes, this is typically done when over-riding a value declared in a bidi further up the cascade, I will verify this and add notes in all the places this occurs :)